ql/src/test/queries/clientpositive/vector_windowing_row_number.q

```diff
@@ -6,13 +6,14 @@ set hive.fetch.task.conversion=none;
 
```

Comment:
please include at least "EXPLAIN VECTORIZATION DETAIL" of a query that utilizes a CastLongToString vectorization should print partition expressions as well I think, in the form of:
```
partitionExpressions: [ConstantVectorExpression(val 0) -> 4:int]
```

Reply:
Do you mean, for each "explain select..." in this file, change to: "EXPLAIN VECTORIZATION DETAIL select..."?

Reply:
I think you should rather use both, keep original and add a new one with VECTORIZATION DETAIL